src/main/java/com/teksystems/Capstone3BackEnd/Models/fff.java

```diff
@@ -1,7 +0,0 @@
-package com.teksystems.Capstone3BackEnd.Models;
-
-public class fff {
```

Comment:
This is a beautiful class.  I think it stays.

Reply:
No, that is gone.